tensorflow/tensorflow.bzl

```diff
@@ -2653,6 +2653,37 @@ def gpu_py_test(
 def cuda_py_test(*args, **kwargs):
     gpu_py_test(*args, **kwargs)
 
+base_tags = [
+        "no_cuda_on_cpu_tap",
+        "no_rocm",
+        "no_windows",
+        "nomac",
+    ]
+
+def trt_py_test(test_names, tags = base_tags):
+    # cuda_py_test enable XLA tests by default. We can't
+    # combine XLA with TensorRT currently and should set
+    # xla_enable_strict_auto_jit to False to disable XLA tests.
```

Comment:
Please reflow the comment to fill the lines except for the last line.

Reply:
Done